linker/Mono.Linker.Steps/BlacklistStep.cs

```diff
@@ -46,7 +46,7 @@ protected override bool ConditionToProcess()
 
 		protected override void Process ()
 		{
-			foreach (string name in Assembly.GetExecutingAssembly ().GetManifestResourceNames ()) {
+			foreach (string name in typeof (BlacklistStep).GetTypeInfo ().Assembly.GetManifestResourceNames ()) {
```

Comment:
Can you please check with @terrajobst regarding all of the appi's you had to change? I believe they should all be available in 2.0.